spring-cloud-aws-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -18,11 +18,16 @@
       "description": "Cloud watch namespace.",
       "type": "java.lang.String"
     },
+    {
```

Comment:
There is no need to add it to `additional-spring-configuration-metadata` as the metadata will be generated from `@ConfigurationProperties` class.

Reply:
Good point, thanks. Reverted.